pkg/transformer/openshift/openshift.go

```diff
@@ -65,6 +65,9 @@ type OpenShift struct {
 // used when undeploying resources from OpenShift
 const TIMEOUT = 300
 
+//default value of Persistent Volume Claim
+const PVCValue = "100Mi"
```

Comment:
do we need separate value for openshift? 
It would be better to reuse `kubernetes.PVCValue`